README.md

```diff
@@ -258,11 +253,24 @@ Parameters:
 
 Returns:
 
--   The result(s) of the call to `callback`
+- The result(s) of the call to `callback`
+
+### Path Manipulation Functions
+
+This library also includes wrappers around the following functions from the [filepath](https://hackage.haskell.org/package/filepath-1.4.2.1/docs/System-FilePath.html) library:
 
+- `take_directory (filepath)` wraps [System.FilePath.takeDirectory](https://hackage.haskell.org/package/filepath-1.4.2.1/docs/System-FilePath-Posix.html#v:takeDirectory)
```

Comment:
These docs will be targeted at Lua users who may not even be aware of Haskell. There should be enough details for anybody to use the functions. Best to use the other docs as reference.

Reply:
Will add the change later